presto-spi/src/main/java/com/facebook/presto/spi/relation/row/VariableReferenceExpression.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.sql.relational;
+package com.facebook.presto.spi.relation.row;
```

Comment:
package `spi.relation` is good enough?

Reply:
OK, forget we are not going to add table level expression but convert planNode instead...